src/v/config/configuration.h

```diff
@@ -372,7 +373,7 @@ struct configuration final : public config_store {
     property<std::vector<ss::sstring>> audit_excluded_principals;
 
     // Archival storage
-    property<bool> cloud_storage_enabled;
+    enterprise<property<bool>> cloud_storage_enabled;
     property<bool> cloud_storage_enable_remote_read;
     property<bool> cloud_storage_enable_remote_write;
```

Comment:
These two?

Reply:
good question. it's not called out explicitly in the requirements but could offer some simplifying assumptions to CS sanctions maybe. besides which it's basically free.

cc @mattschumpert @michael-redpanda thoughts?

Reply:
from standup: any change here pending Ben's investigation on the CS sanctions PR